scripts/metavar.py

```diff
@@ -771,9 +784,21 @@ def intrinsic_elements(self, check_dict=None):
         Fortran does not support a way to reference those elements.
         """
         if self.is_ddt():
-            element_names = None
-            raise ValueError("shouldn't happen?")
-            # To Do, find and process named elements of DDT
+            dtitle = self.get_prop_value('type')
```

Comment:
I'm confused. It seems that you can get out of this block without `element_names` being defined but the method returns `element_names`. If it is somehow 'impossible', I would at least raise an `Exception` at the points where the logic allows an exit without setting `element_names`.